src/solitaire.jl

```diff
@@ -1,4 +1,4 @@
-include("common.jl")
+# include("common.jl")
```

Comment:
Could it be because ClassicalCiphers.jl does `include("common.jl")`? I don't know how Julia scopes includes, and I'm really tired right now :P